api/hypercore/add_columnstore_policy.md

```diff
@@ -0,0 +1,108 @@
+
+<Since2180 />
+
+## Samples
+
+To create a columnstore job:
+
+<Procedure>
+
+1. **Enable columnstore**
+
+   * [Use `ALTER TABLE` for a hypertable][compression_alter-table]
+     ```sql
+     ALTER TABLE stocks_real_time SET (timescaledb.enable_columnstore = true, timescaledb.segmentby = 'symbol');
+     ```
+   * [Use ALTER MATERIALIZED VIEW for a continuous aggregate][compression_continuous-aggregate]
+     ```sql
+     ALTER MATERIALIZED VIEW stock_candlestick_daily set (timescaledb.enable_columnstore = true, timescaledb.segmentby = 'symbol' );
+     ```
+
+1. **Add a policy to move chunks to the columnstore at a specific time interval**
+
+   For example:
+
+   * 60 days after the data was added to the table:
+     ``` sql
+     CALL add_columnstore_policy('stocks_real_time', after => INTERVAL '60d');
+     ```
+   * 3 months prior to the moment you run the query:
+
+     ``` sql
+     CALL add_columnstore_policy('stocks_real_time', created_before => INTERVAL '3 months');
+     ```
+   * With an integer-based time column:
+
+     ``` sql
+     CALL add_columnstore_policy('table_with_bigint_time', BIGINT '600000');
+     ```
+   * Older than eight weeks:
+
+     ``` sql
+     CALL add_columnstore_policy('cpu_weekly', INTERVAL '8 weeks');
+     ```
+     
+1. **View the policies that you set or the policies that already exist** 
+
+   ``` sql
+   SELECT * FROM timescaledb_information.jobs
+   WHERE proc_name='policy_compression';
+   ```
+   See [timescaledb_information.jobs][informational-views].
+
+</Procedure>
+
+## Arguments
+
+Calls to `add_columnstore_policy` require either `after` or `created_before`, but cannot have both.
+
+<!-- vale Google.Acronyms = NO -->
+<!-- vale Vale.Spelling = NO -->
+
+| Name | Type | Default | Required | Description |
+|--|--|--|--|--|
+| `hypertable`              |REGCLASS| - | ✔ | Name of the hypertable or continuous aggregate to run this [job][job] on.|
+| `after`          |INTERVAL or INTEGER|- | ✖ | Add chunks containing data older than `now - {after}::interval` to the columnstore. <br/> Use an object type that matchs the time column type in `hypertable`: <ul><li><b><code>TIMESTAMP</code>, <code>TIMESTAMPTZ</code>, or <code>DATE</code></b>: use an <code>INTERVAL</code> type.</li><li><b> Integer-based timestamps </b>: set an integer type using the [integer_now_func][set_integer_now_func].</li></ul> `after` is mutually exclusive with `created_before`. |
+| `created_before` |INTERVAL| NULL | ✖ | Add chunks with a creation time of `now() - created_before` to the columnstore. <br/> `created_before` is <ul><li>Not supported for continuous aggregates.</li><li>Mutually exclusive with `after`.</li></ul> |
+| `schedule_interval`       |INTERVAL| 12 hours when [chunk_time_interval][chunk_time_interval] >= `1 day` for `hypertable`. Otherwise `chunk_time_interval` / `2`. | ✖        | Set the interval between the finish time of the last execution of this policy and the next start.|
```

Comment:
This is not required:
```suggestion
| `schedule_interval`       |INTERVAL| 12 hours when [chunk_time_interval][chunk_time_interval] >= `1 day` for `hypertable`. Otherwise `chunk_time_interval` / `2`. |         | Set the interval between the finish time of the last execution of this policy and the next start.|
```